lib/solargraph/workspace/config.rb

```diff
@@ -13,7 +13,7 @@ class Config
       def initialize workspace = nil
         @workspace = workspace
         include_globs = ['**/*.rb']
-        exclude_globs = ['spec/**/*', 'test/**/*', 'vendor/**/*']
+        exclude_globs = ['spec/**/*', 'test/**/*', 'vendor/**/*', '.bundle/**/*']
```

Comment:
This is a bit of a personal touch. WDYT, @castwide?

Reply:
Looks good to me. As a general rule, Solargraph should exclude folders that contain gem caches.